datahub-web-react/src/app/auth/ResetCredentials.tsx

```diff
@@ -40,6 +40,13 @@ const FormInput = styled(Input)`
     }
 `;
 
+const StyledFormItem = styled(Form.Item)`
+    .ant-input-affix-wrapper-status-error:not(.ant-input-affix-wrapper-disabled):not(.ant-input-affix-wrapper-borderless).ant-input-affix-wrapper,
+    .ant-input-affix-wrapper-status-error:not(.ant-input-affix-wrapper-disabled):not(.ant-input-affix-wrapper-borderless).ant-input-affix-wrapper:hover {
+        background-color: transparent;
+    }
```

Comment:
woah haha is there a simpler way to set this background color?

Reply:
It has been simplified!